systemtest/src/test/java/io/strimzi/systemtest/olm/OlmAllNamespaceST.java

```diff
@@ -21,52 +27,129 @@
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @Tag(OLM)
+@SuiteDoc(
+    description = @Desc("Tests Strimzi deployments managed by OLM when configured to watch all namespaces."),
+    labels = {
+        @Label(TestDocsLabels.OLM)
+    }
+)
 public class OlmAllNamespaceST extends OlmAbstractST {
 
     public static final String NAMESPACE = "olm-namespace";
 
     @Test
     @Order(1)
+    @TestDoc(
+        description = @Desc("Verifies the deployment of a Kafka cluster using the OLM example when the operator watches all namespaces."),
+        steps = {
+            @Step(value = "Deploy Kafka cluster using the OLM example.", expected = "Kafka cluster is deployed and becomes ready."),
+            @Step(value = "Wait for readiness of the Kafka cluster, meaning that the operator (watching all namespaces) manages the Kafka cluster in the designated test namespace.", expected = "Kafka cluster is operational and managed by the operator.")
```

Comment:
Sounds to me like both are doing the same. I think the first ste should be something like `Create Kafka CR -> Kafka CR is created in Kubernetes`.